data/spells/scripts/attack/poison_storm.lua

```diff
@@ -4,10 +4,26 @@ combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_GREEN_RINGS)
 
 combat:setArea(createCombatArea(AREA_CIRCLE6X6))
 
+local condition = createConditionObject(CONDITION_POISON)
+setConditionParam(condition, CONDITION_PARAM_DELAYED, 1)
```

Comment:
please use tfs 1.x equivalents. This is old pre 1.x coding style.
setConditionParam is for example condition:setParameter
addDamageCondition is condition:addDamage